src/_locales/zh/messages.json

```diff
@@ -351,6 +351,9 @@
     "service_decryption_failed": {
         "message": "解密失敗。"
     },
+    "service_self_key_decryption_failed": {
+        "message": "解密失敗。 或許您選取了錯誤的檔案或導入了錯誤的金鑰。"
```

Comment:
Extra space character here.

Reply:
it's a change from another branch, resolved in that branch